CMakeLists.txt

```diff
@@ -526,4 +526,31 @@ else ()
                         "${GTEST_SOURCE_DIR}/googletest/include")
     target_link_libraries(draco_tests draco gtest)
   endif ()
+
+  # Headers installation
+  file ( GLOB_RECURSE draco_headers RELATIVE ${CMAKE_SOURCE_DIR} "*.h" )
+  foreach (filename ${draco_headers})
+    get_filename_component ( file_directory ${filename} DIRECTORY )
+    install ( FILES
```

Comment:
ditto on collapsing horizontal whitespace, same for the following commands. The foreach and endforeach are the only commands in here that should have a space between the command and the open parenthesis given the style throughout the rest of the cmake file.